example/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-add_custom_target(examples DEPENDS examples)
+add_custom_target(examples DEPENDS examplesae)
```

Comment:
I believe it allows `make examples` to detect that it needs to be re-compiled if source in the `example/` directory changes.